lib/enumerable/none.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_none?(pattern = nil, &a_block)
+    !my_any?(pattern, &a_block)
+  end
```

Comment:
`[1, 3.14, 42].none?(Float)                         #=> false`

Reply:
This was already taken care of

![image](https://user-images.githubusercontent.com/36519478/68625397-08170f80-04a7-11ea-90d9-f3734ab6ebb4.png)